lib/LaTeXML/Package/expl3.lua.ltxml

```diff
@@ -0,0 +1,124 @@
+      if (!$table) {                            # Create new table, if none
+        $table = [];
+        AssignValue($address => $table, 'global'); }
+      return $table; }
+    else {
+      $gullet->unread($tok);
+      Error('expected', '__intarray:w', $gullet,
+        "Expected an intarray identifier, got " . ToString($tok));
+      return []; } });
+
+DefPrimitiveI('\__intarray:w', 'Number', sub {
+    Error('unexpected', '\__intarray:w', $_[0], "Unexpected isolated \\__intarray:w?");
+    return; },
+  protected => 1);
+
+DefPrimitiveI('\__intarray_gset_count:Nw', 'Intarray Number', sub {
+    my ($stomach, $table, $newlength) = @_;
+    my $length = scalar(@$table);
+    $newlength = $newlength->valueOf;
+    $$table[$newlength - 1] = 0;                     # Grow in one step for efficiency
+    for (my $i = $length + 1 ; $i <= $newlength ; $i++) {
+      $$table[$i - 1] = 0; }
+    return; },
+  protected => 1);
+
+DefMacroI('\intarray_count:N', 'Intarray', sub {
+    my ($gullet, $table) = @_;
+    return Explode(scalar(@$table)); });
+
+DefMacroI('\__intarray_gset:wF', 'Number Intarray Number {}', sub {
+    my ($stomach, $pos, $table, $value, $ifmissing) = @_;
+    my $length = scalar(@$table);
+    $pos = $pos->valueOf;
+    if (($pos > 0) && ($pos < $length)) {
+      $$table[$pos - 1] = $value->valueOf; }
+    else {
+      return $ifmissing; }
+    return; },
+  protected => 1);
+
+DefMacroI('\__intarray_gset:w', 'Number Intarray Number', sub {
+    my ($stomach, $pos, $table, $value) = @_;
+    my $length = scalar(@$table);
+    $pos = $pos->valueOf;
+    if ($pos > 0) {
+      $$table[$pos - 1] = $value->valueOf; }
+    return; },
+  protected => 1);
+
+DefPrimitiveI('\intarray_gzero:w', 'Intarray', sub {
+    my ($stomach, $table) = @_;
+    my $length = scalar(@$table);
+    for (my $i = 1 ; $i <= $length ; $i++) {
+      $$table[$i - 1] = 0; }
+    return; },
+  protected => 1);
+
+DefMacroI('\__intarray_item:wF', 'Number Intarray {}', sub {
+    my ($stomach, $pos, $table, $ifmissing) = @_;
+    my $length = scalar(@$table);
+    $pos = $pos->valueOf;
+    if (($pos > 0) && ($pos < $length)) {
+      return Explode($$table[$pos - 1]); }
+    else {
+      return $ifmissing; } });
+
+DefMacroI('\__intarray_item:w', 'Number Intarray', sub {
+    my ($stomach, $pos, $table) = @_;
+    my $length = scalar(@$table);
+    $pos = $pos->valueOf;
+    if (($pos > 0) && ($pos < $length)) {
+      return Explode($$table[$pos - 1]); }
+    return; });
+
+DefMacroI('\__intarray_to_clist:Nn', 'Intarray', sub {
+    my ($stomach, $table) = @_;
+    return Tokenize(join(', ', @$table)); });
+
+DefMacroI('\__intarray_range_to_clist:w', 'Intarray Number Number', sub {
+    my ($stomach, $table, $from, $to) = @_;
+    $from = $from->valueOf - 1;
+    $to   = $to->valueOf - 1;
+    return Tokenize(join(', ', @$table[$from .. $to])); });
+
+DefPrimitiveI('\__intarray_gset_range:w', 'Number Intarray', sub {
```

Comment:
Since `\__kernel_intarray_gset_range_from_clist:Nnn` calls `\__intarray_gset_range:w \int_eval:w #2 #1 #3 , , \scan_stop:`, I think this will leave `#3 , , \scan_stop:` in the token stream.  Should this be something like
```perl
DefPrimitiveI('\__intarray_gset_range:w',
  LaTeXML::Core::Parameters->new(
    LaTeXML::Core::Parameter->new('Number'),
    LaTeXML::Core::Parameter->new('Intarray'),
    LaTeXML::Core::Parameter->new('Until','Until:\scan_stop:',extra=>[T_CS('\scan_stop:')]) ),
  sub {
    ... } );
```
On the other hand, there doesn't appear to be a public version of the function, so I'm not sure how often this will get called (also, being unimplemented will throw an error anyway).